mx.truffle/suite.py

```diff
@@ -364,6 +364,21 @@
        "workingSets" : "Truffle,SimpleLanguage,Tools",
        "license" : "UPL",
      },
+
+    "com.oracle.truffle.examples" : {
+       "subDir" : "truffle",
```

Comment:
I don't think we need new project/module. Rather than a static example, it is better to convert the sample into test that we execute regularly. I would put it into one of existing `*.test` projects. Creating new *test* project is also an option, but I am not sure we need that.

Please note that https://github.com/graalvm/truffle/pull/126/ tries to introduce micro benchmarks into Truffle and this example is a perfect candidate for a microbenchmark.